apps/scripts/main.py

```diff
@@ -47,7 +49,7 @@ def call_external(script_list, shell=False):
             Printer(["Unknown format,", "forgot header?"], i, o, 1)
         else:
             error_data = ["Unknown error", ""]
-            error_data += ffs(repr(e), o.cols)
```

Comment:
Lost a feature there - Printer will now print out an unformatted exception message string as the third line (so, likely, will not print anything readable at all, as Printer by itself doesn't wrap strings).